osu.Game/Overlays/Settings/Sections/General/InstallationSettings.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework;
+using osu.Framework.Allocation;
+using osu.Framework.Localisation;
+using osu.Framework.Platform;
+using osu.Framework.Screens;
+using osu.Game.Localisation;
+using osu.Game.Overlays.Settings.Sections.Maintenance;
+
+namespace osu.Game.Overlays.Settings.Sections.General
+{
+    public partial class InstallationSettings : SettingsSubsection
+    {
+        protected override LocalisableString Header => GeneralSettingsStrings.InstallationHeader;
+
+        [Resolved]
+        private OsuGame? game { get; set; }
+
+        [BackgroundDependencyLoader]
+        private void load(Storage storage)
+        {
+            bool isDesktop = RuntimeInfo.IsDesktop;
+
+            // Loosely update-related maintenance buttons.
+            if (isDesktop)
```

Comment:
May wanna pull this `isDesktop` check up a level to `GeneralSection` such that mobile doesn't see a stupid-looking empty section.